bases/br_sp_seduc_fluxo_escolar/municipio/publish.sql

```diff
@@ -0,0 +1,39 @@
+/*
+
+Query para publicar a tabela.
+
+Esse é o lugar para:
+    - modificar nomes, ordem e tipos de colunas
+    - dar join com outras tabelas
+    - criar colunas extras (e.g. logs, proporções, etc.)
+
+Qualquer coluna definida aqui deve também existir em `table_config.yaml`.
+
+# Além disso, sinta-se à vontade para alterar alguns nomes obscuros
+# para algo um pouco mais explícito.
+
+TIPOS:
+    - Para modificar tipos de colunas, basta substituir STRING por outro tipo válido.
+    - Exemplo: `SAFE_CAST(column_name AS NUMERIC) column_name`
+    - Mais detalhes: https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types
+
+*/
+
+CREATE VIEW basedosdados-dev.br_sp_seduc_fluxo_escolar.municipio AS
+SELECT 
+SAFE_CAST(ano AS STRING) ano,
+SAFE_CAST(id_municipio AS INT64) id_municipio,
+SAFE_CAST(municipio AS STRING) municipio,
+SAFE_CAST(prop_aprovados_anos_inciais_ef AS STRING) prop_aprovados_anos_inciais_ef,
+SAFE_CAST(prop_reprovados_anos_iniciais_ef AS STRING) prop_reprovados_anos_iniciais_ef,
+SAFE_CAST(prop_abandono_anos_iniciais_ef AS STRING) prop_abandono_anos_iniciais_ef,
+SAFE_CAST(prop_aprovados_anos_finais_ef AS STRING) prop_aprovados_anos_finais_ef,
+SAFE_CAST(prop_reprovados_anos_finais_ef AS STRING) prop_reprovados_anos_finais_ef,
+SAFE_CAST(prop_abandono_anos_finais_ef AS STRING) prop_abandono_anos_finais_ef,
+SAFE_CAST(prop_aprovados_em AS STRING) prop_aprovados_em,
+SAFE_CAST(prop_reprovados_em AS STRING) prop_reprovados_em,
+SAFE_CAST(prop_abandono_em AS STRING) prop_abandono_em,
+SAFE_CAST(codigo_rede_ensino AS STRING) codigo_rede_ensino,
+SAFE_CAST(rede_ensino AS STRING) rede_ensino,
+SAFE_CAST(diretoria AS STRING) diretoria
+from basedosdados-dev.br_sp_seduc_fluxo_escolar_staging.municipio as t
```

Comment:
Faltou editar aqui.

Reply:
Talvez seja melhor colocar `codigo_rede_ensino` `rede_ensino` e `diretoria` no início.

Reply:
Concordo